Skip this test, which uncovers a race condition in how 'tail' handles
file deletion.  See <http://bugs.gnu.org/21460>.

--- a/tests/tail-2/assert.sh
+++ b/tests/tail-2/assert.sh
@@ -16,6 +16,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+exit 77				# see <http://bugs.gnu.org/21460>
 
 # This test fails with tail from textutils-2.0.
 # It would get something like this:
